actions/src/openstack_check_actions.py

```diff
@@ -0,0 +1,447 @@
+import json
+import os
+from typing import Callable
+import logging
+import sys
+from datetime import datetime
+import requests
+from openstack_api.openstack_connection import OpenstackConnection
+from openstack_action import OpenstackAction
+from requests.auth import HTTPBasicAuth
+
+
+class CheckActions(OpenstackAction):
```

Comment:
You want to inherit from `Action` rather than `OpenstackAction` (which is the legacy API)
This means we're using Stackstorm as-is and we can simply use their docs in the future too